lib/helpers/deprecated-rules.js

```diff
@@ -4,6 +4,7 @@ let DEPRECATED_RULES = new Map([
   // deprecated in the 1.x cycle, can be removed in 3.x
```

Comment:
Since we're on version 2.8, should we add a new comment saying `// deprecated in the 2.x cycle, can be removed in 4.x` before the new addition?

Reply:
Added a note.